spec/unit/bootstrap/definitions_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe FHIR::Definitions do
+  context '.load_extensions'
+
+  context '.get_display' do
+    let(:code) { '306005' }
+    let(:uri) { 'http://snomed.info/sct' }
```

Comment:
I agree, these seem like a good testing starting point. Perhaps we should find one that will hit the `expansion` branches, once we get those sorted out.